bokeh/models/axes.py

```diff
@@ -26,10 +26,6 @@ class Axis(GuideRenderer):
     Ability to hide the entire axis from the plot.
     """)
 
-    location = Either(Auto, Enum(Location), help="""
-    Where should labels and ticks be located in relation to the axis rule.
-    """)
-
```

Comment:
OK but we should mark the property deprecated and add a no-op (python) property getter and setter so that trying to use `.location` doesn't just fall over with an exception